wordle-web/pages/index.vue

```diff
@@ -12,6 +12,11 @@
                 Play
               </v-btn>
             </v-card-actions>
+            <v-card-actions class="justify-center">
+              <v-btn nuxt to="/game" color="primary" x-large width="200px">
```

Comment:
Still just links to the game page, not an implementation of the word of the day. Missing the word of the day page.

Reply:
Looks like this is still unresolved. Also looks like you guys didn't implement any of the API portions for this project.